src/components/sidebar/routePathView/routePathInfoTab/RoutePathForm.tsx

```diff
@@ -59,12 +53,33 @@ class RoutePathForm extends React.Component<IRoutePathFormProps>{
         window.alert('Toteutuksen suunnittelu kesken.');
     }
 
+    private renderLengthLabel = () => {
+        const isEditingDisabled = this.props.isEditingDisabled;
+        const updateLength = () => {
```

Comment:
Why was updateLength implementation moved in to the function? This will force the recreating of the function, every time we render the label. I would recommend extracting it.

Reply:
I fixed this

Reply:
I think fixing outside of function is only necessary if we have a list item or some other case where the component is rendered multiple times.

But as readability here is not that critical, go with how it is now. In the future I wouldn't mind declaring functions in render if that render function is not going to be called multiple times since its only a minor optimization